flask_admin/helpers.py

```diff
@@ -9,6 +9,8 @@
 
 
 VALID_SCHEMES = ['http', 'https']
+_substitute_whitespace = compile(r'[\s\x00-\x08\x0B\x0C\x0E-\x19]+').sub
```

Comment:
are there tests for all these specific ASCII characters that we are matching against? The only whitespace that I see in the tests are in `assert not helpers.is_safe_url(' javascript:alert(document.domain)')` and for that use-case a simple `replace(' ', '') would do the job
Please either expand the tests, or simplify the logic

Reply:
P.S. when checking for control characters, wouldn't it be simpler to do it in the same way as it's done in Django by checking `unicodedata.category` (see https://github.com/django/django/blob/550cb3a365dee4edfdd1563224d5304de2a57fda/django/utils/http.py#L364)?